README.md

```diff
@@ -1 +1,33 @@
-Terraform configuration for kafka clusters to create topis and ACLs
+# Terraform configuration for kafka clusters to create topis and ACLs
+
+## how to plan/ apply shared Kafka Terraform resources?
+
+#### Prerequisities
+- [Terraform](https://developer.hashicorp.com/terraform/install) installed
+- [kubefwd](https://github.com/txn2/kubefwd/releases) installed
+- Following roles should exists in the .aws/credentials file:
+```
+[pubsub-dev]
+region         = eu-west-1
+role_arn       = arn:aws:iam::950135041896:role/pubsub-admin
+source_profile = uw-prod-jump-role
+
+[pubsub-prod]
+region         = eu-west-1
+role_arn       = arn:aws:iam::703452047160:role/pubsub-admin
+source_profile = uw-prod-jump-role
+```
+
+
+#### Run the following commands to plan/apply the changes
+
+1. Reference AWS profile necessary to perform next actions- <br> `export AWS_PROFILE=pubsub-dev` <br> or <br> `export AWS_PROFILE=pubsub-prod` <br> in each terminal window.
```

Comment:
not sure all the line breaks add much here? These feels as readable on one-line as split across many (but reads a bit messier with all the `<br>`)